### PR TITLE
add new translation on r

### PR DESCRIPTION
**Pull request by arojascastro**
Attempting to add a new translation by @jenniferisasi 

Please approve after checking TRAVIS validation process.

### CODE REVIEW

**Comment by arojascastro** [comment]
Ok, all checks passed!

**Comment by arojascastro** [comment]
@mdlincoln I'm sorry for bothering again, but I cannot see where is the problem. I get this warning:

```- ./_site/es/lecciones/index.html
  *  internal image /gallery//en/basic-text-processing-in-r.png does not exist (line 1018)
htmlproofer 3.9.0 | Error:  HTML-Proofer found 1 failure!
```

But I do not find `/basic-text-processing-in-r.png` in the file.

**Comment by acrymble** [comment]
you have 2 slashes after /gallery

**Comment by arojascastro** [comment]
But I do not find that `/gallery//en/basic-text-processing-in-r.png` in the text...

**Comment by walshbr** [comment]
Looks like the error is coming up on a different page - the lessons index - based on the metadata for the lesson you added. I think it's this line from your metadata in the header of the file you added - line 20:

`original: /en/basic-text-processing-in-r`

Copying the format from the other Spanish lessons it looks like it should be -

`original: basic-text-processing-in-r`

Looks like it just takes the value of that key and tries to find the image in the galleries folder from that. But there's no 'en' folder in our gallery folder. So when you added the new lesson it is trying to find the image for the lesson but getting tripped up. Let me know if that doesn't make sense!

**Comment by arojascastro** [comment]
Thank you @walshbr - that was the error! :)

**Comment by arojascastro** [comment]
@jenniferisasi I had to change some code containing a link:

From this


```{r}
base_url <- "https://programminghistorian.org/assets/basic-text-processing-in-r"
url <- sprintf("%s/sotu_text/236.txt", base_url)
texto <- paste(readLines(url), collapse = "\n")
```


To this 

```
+{r}
+base_url <- "/assets/basic-text-processing-in-r"
+url <- sprintf("%s/sotu_text/236.txt", base_url)
+texto <- paste(readLines(url), collapse = "\n")
+
```

Does it make sense to you? I guess you followed the original, but the new TRAVIS requirements forced me to make the path relative. Does still work in the context of the lesson? I hope.

**Comment by mdlincoln** [comment]
Oh, no, the full link `https://programminghistorian.org/assets/basic-text-processing-in-r` should be ok when it's in the context of a code block. I thought I had adjusted the Travis checks to ignore those 😠  Give me a few minutes to check out why it was being triggered.

**Comment by arojascastro** [comment]
Well maybe it was not alerting and I changed it because I thought that lied
the problem. I can undo and double check

-- 
​Dr. Antonio Rojas Castro
Researcher, Cologne Center for eHumanities
Communication coordinator, EADH
<http://www.antoniorojascastro.com>

El día 22 de junio de 2018 a las 16:47:18, Matthew Lincoln (
notifications@github.com) escrito:

**Comment by mdlincoln** [comment]
Yes, it looks like you changed two links in commit ad8e2a9560dc478b655909fa1d719c428932f0cf. The one in the R code was fine - it was only the one in the markdown text that was triggering the alert. I can try to update the code check to try and show the line number or context of the problem links so that it's easier to understand where the errors are.

**Comment by arojascastro** [comment]
Thank you all! The full link was restored by Matthew. If Travis is not arguing, can anyone approve the pull request please?

**Comment by mdlincoln** [comment]
hrm, not yet - there is a redirect loop going on. Let me check the lesson metadata again

**Comment by mdlincoln** [comment]
I've pushed fixes for a few errors:

- We don't use `redirect_from` in translated lessons anymore
- Translated images still go into the image directory of the original lesson, so I moved the images into `/images/basic-text-processing-in-r`
- When using `{% incldue figure.html...` only put the name of the file, no other path needed.

File: es/lecciones/procesado-basico-de-textos-en-r.md [hunk 1]
```diff
@@ -0,0 +1,628 @@
...
+* ajustar modelos temáticos (*topic models*) para detectar discursos particulares en el corpus usando paquetes como **mallet**[^12] y **topicmodels**[^13].
+* aplicar técnicas de reducción dimensional para crear gráficos de tendencias estilísticas a lo largo del tiempo o entre múltiples autores. Por ejemplo, el paquete **tsne**[^14] realiza una forma poderosa de reducción dimensional particularmente apta para gráficos detallados.
+
+Existen muchos tutoriales genéricos para estos tres ejemplos, además de documentación detallada de los paquetes[^15]. Esperamos ofrecer tutoriales enfocados en aplicaciones históricas en particular en el futuro.
+
+# Notas
+
+[^1]: Nuestro corpus contiene 236 discursos del Estado de la Unión. Dependiendo de lo que se cuente, este número puede ser ligeramente más alto o más bajo.
+[^2]: Taryn Deward, [“Datos tabulares en R,” Programming Historian](/es/lecciones/datos-tabulares-en-r). Trad. de Jennifer Isasi de 10 de abril de 2018.
```
Comment (mdlincoln) [inline es/lecciones/procesado-basico-de-textos-en-r.md]: This translation does not appear to be published yet?

**Review by mdlincoln — CHANGES_REQUESTED** [review]

File: es/lecciones/procesado-basico-de-textos-en-r.md (hunk 1 above)
Comment (arojascastro) [reply es/lecciones/procesado-basico-de-textos-en-r.md]: It's currently under review: https://github.com/programminghistorian/ph-submissions/issues/164

I'm hoping to publish it in the following days. But yes we shouldn't linked it yet...

**Comment by arojascastro** [comment]
I removed that internal link and now it looks all ok.

**Review by mdlincoln — DISMISSED** [review]

**Comment by mdlincoln** [comment]
OK @arojascastro you might want to open an issue to remember to add the link back in when “Datos tabulares en R” in eventually published.

**Comment by arojascastro** [comment]
I changed the name file and title from "procesado" to "procesamiento". I hope @jenniferisasi agrees.

**Comment by arojascastro** [comment]
Sorry, I need someone to review and accept this merge :) thanks

**Review by mdlincoln — APPROVED** [review]
I'm happy if @programminghistorian/spanish-team is happy 😄

**Comment by mdlincoln** [comment]
Merge whenever you're ready @arojascastro

**Pull request merged**